include/taskolib/Sequence.h

```diff
@@ -265,6 +272,13 @@ class Sequence
      */
     void execute(Context& context, CommChannel* comm_channel);
 
+    /**
+     * Get the step setup script object.
+     *
+     * @return const std::string& step setup script.
```

Comment:
Oh, nice! With `\returns` it sounds more like natural pronunciation than its _artificial_ `\return`.  Changed.

Note: VS Code produces an automatic created template (at least my VS Code), filling the required Doxygen commands from the function signature with some meaningful content. And it also uses the `@`-prefix of the commands. I need to tweak the Doxygen config...

Reply:
As far as I understand `@` or `\` is just a matter of taste. Both are fine. We should just try to avoid changing style too often in a single source file... :)